VideoWeb/VideoWeb.Common/Models/ConsultationInvitation.cs

```diff
@@ -0,0 +1,66 @@
+﻿using System;
+using System.Collections.Concurrent;
+using System.Collections.Generic;
+using System.Linq;
+using Newtonsoft.Json;
+
+namespace VideoWeb.Common.Models
+{
+    public interface IConsultationInvitation
+    {
+        Guid InvitationId { get; }
+        Guid RequestedForParticipantId { get; }
+        ConcurrentDictionary<Guid, ConsultationAnswer> InvitedParticipantResponses { get; }
+        bool HaveAllAccepted { get; }
+        bool HaveAllResponded { get; }
+        bool HasSomeoneRejected { get; }
+    }
+
+    public class ConsultationInvitation : IConsultationInvitation
+    {
+        [JsonConstructor]
```

Comment:
this ctor doesn't seem to be doing anything special that would requires this attribute

Reply:
It is used for the serialisation into JSON for the cache. I think I could just add the default constructor instead? Not sure I would have to mark it.